### ISSUE TITLE
Update readme to reflect actual commands for pulling correct Docker image

### ISSUE THREAD

**Issue by tummyache123** [ellitedev/srb2kart-server-docker#4]
Currently, the readme reflects jetcodesstuff's srb2kart repo, which when the `docker run -it --name srb2kart -p 5029:5029/udp jetcodesstuff/srb2-kartserv:latest` command is executed, will return with an error that the architecture doesn't match (expected arm, got x86_64), which I imagine most people in the community are using an Ubuntu VPS that runs off an x84_64 CPU, myself included. `docker run -it --name srb2kart -p 5029:5029/udp ellite/srb2kart-server:latest` works as expected.

It would probably be a good idea to update the readme to match the instructions posted on the srb2kart message board, especially seeing as the post there has a direct link to your repo.

Not related to this issue and is probably out of scope for you, but having a better guide for others who are beginners to Linux and looking for a quick and easy way to host a kart server would be helpful, especially with regard to setting up addons. It's something I'd like to tackle myself, but only once I've managed to successfully get everything working and have a complete understanding of how it all works.

Thank you for your work on this container.
